<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExportTarDialog</class>
 <widget class="QDialog" name="ExportTarDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Export to Tarball</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_info">
     <property name="text">
      <string>Export the selected archive to a tarball.</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_archive">
       <property name="text">
        <string>Archive:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="archiveNameLabel">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>archive_name</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_dest">
       <property name="text">
        <string>Destination:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_dest">
       <item>
        <widget class="QLineEdit" name="destinationInput">
         <property name="placeholderText">
          <string>Path to save the .tar file</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnBrowse">
         <property name="text">
          <string>Browse...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_compress">
       <property name="text">
        <string>Compression:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="comboCompression">
       <item>
        <property name="text">
         <string>none</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>gzip</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>lz4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>zstd</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>xz</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_strip">
       <property name="text">
        <string>Strip Components:</string>
       </property>
       <property name="toolTip">
        <string>Remove the specified number of leading path elements.</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSpinBox" name="spinStripComponents">
       <property name="minimum">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_excludes">
       <property name="text">
        <string>Exclude Patterns:</string>
       </property>
       <property name="toolTip">
        <string>Space-separated list of exclude patterns.</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="inputExcludes">
       <property name="placeholderText">
        <string>e.g. *.tmp home/user/.cache</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_paths">
       <property name="text">
        <string>Specific Paths:</string>
       </property>
       <property name="toolTip">
        <string>Space-separated list of paths to extract.</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="inputPaths">
       <property name="placeholderText">
        <string>Leave empty to export all</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_format">
       <property name="text">
        <string>Tar Format:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QComboBox" name="comboFormat">
       <item>
        <property name="text">
         <string>GNU</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>PAX</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ExportTarDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ExportTarDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
